src/test/java/core/basesyntax/dao/FruitTransactionDaoImplTest.java

```diff
@@ -0,0 +1,175 @@
+package core.basesyntax.dao;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+class FruitTransactionDaoImplTest {
+    private final FruitTransactionDao fruitTransactionDao = new FruitTransactionDaoImpl();
+
+    @AfterEach
+    public void afterEachTest() {
+        Storage.fruits.clear();
+    }
+
+    @Test
+    public void update_severalField_ok() {
+        Storage.fruits.put("banana", 10);
+        Storage.fruits.put("apple", 10);
+        Storage.fruits.put("something", 10);
+        fruitTransactionDao.update("banana", 20);
+
+        Map<String, Integer> nameQuantityMap = new HashMap<>();
+        nameQuantityMap.put("banana", 20);
+        nameQuantityMap.put("apple", 10);
+        nameQuantityMap.put("something", 10);
+
+        assertEquals(Storage.fruits, nameQuantityMap);
+    }
+
+    @Test
+    public void update_notExistField_ok() {
+        Storage.fruits.put("apple", 10);
+        Storage.fruits.put("something", 10);
+        fruitTransactionDao.update("banana", 20);
+
+        Map<String, Integer> nameQuantityMap = new HashMap<>();
+        nameQuantityMap.put("apple", 10);
+        nameQuantityMap.put("something", 10);
+
+        assertEquals(Storage.fruits, nameQuantityMap);
+    }
+
+    @Test
+    public void update_nullField_ok() {
```

Comment:
```suggestion
    public void update_nullField_throwsException() {
```
I don't think that is `ok`, please make sure all test methods are named in correct way